lib/sidekiq/cron/job.rb

```diff
@@ -529,9 +539,17 @@ def self.destroy_all!
 
       # Remove "removed jobs" between current jobs and new jobs
       def self.destroy_removed_jobs new_job_names
-        current_job_names = Sidekiq::Cron::Job.all.map(&:name)
+        current_jobs = Sidekiq::Cron::Job.all('*')
```

Comment:
Existing code looks to use double quotes, should maintain consistent quoting style